mintapi/cli.py

```diff
@@ -375,6 +382,18 @@ def output_data(options, data, attention_msg=None):
                 f.write(attention_msg)
 
 
+def json_to_csv(data, filename):
```

Comment:
Does this only work for specifically formatted "flat" JSON values? If so, I wonder if there should be some exception handling and a useful error message. If not, it's probably fine!

Reply:
@mrooney it is, mainly because this method is only used right now for investment data which is flat. If we end up needing to expand to convert other data sources that may not be flat, we'd probably want to have a flattening routine like we have with credit utilization. 

So I could add a comment calling attention to this fact if that would make it clearer and safer?

Reply:
Yeah, my main concern was that the method name implies you could use it to convert any JSON to CSV. So my only nitpick would be that we should either have a check for flatness / compatibility in there that raises a useful error if it fails, and//or rename the method to something like flat_json_to_csv.